src/Tasks/NETMessage.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Tasks.Deployment.Bootstrapper;
+
+namespace Microsoft.Build.Tasks
+{
+    public sealed class NETMessage : TaskExtension
```

Comment:
Can you please comment the new class, mentioning that the task is intended to be called from internal targets only? Also, I'm wondering if there's a more descriptive name, or if `NetMessage` wouldn't be better aligned with naming conventions.

Reply:
How does "MSBuildMessage" sound to you?